stream/src/main/scala/org/apache/pekko/stream/scaladsl/FlowWithContextOps.scala

```diff
@@ -88,6 +88,20 @@ trait FlowWithContextOps[+Out, +Ctx, +Mat] {
   def viaMat[Out2, Ctx2, Mat2, Mat3](flow: Graph[FlowShape[(Out, Ctx), (Out2, Ctx2)], Mat2])(
       combine: (Mat, Mat2) => Mat3): ReprMat[Out2, Ctx2, Mat3]
 
+  /**
+   * Data variant of [[pekko.stream.scaladsl.FlowOps.alsoTo]]
+   *
+   * @see [[pekko.stream.scaladsl.FlowOps.alsoTo]]
+   */
+  def alsoTo(that: Graph[SinkShape[Out], _]): Repr[Out, Ctx]
```

Comment:
Can you add `@since 1.1.0` on this and other new methods?

Reply:
Will do

Reply:
Added and pushed